__tests__/capitalize.test.js

```diff
@@ -0,0 +1,6 @@
+import capitalized from '../src/capitilaze';
+
+test('capitalize first letter of string', () => {
+  expect(capitalized('safa')).toBe('Safa');
+  expect(capitalized('phillip')).not.toBe('phillip');
```

Comment:
- [ ] [OPTIONAL] You could also add a test for when the first letter already id capitalized.